### DISCUSSION TITLE
Tabi VS Whitty

### DISCUSSION THREAD

**Discussion by DzGithub2805** [luckydog7/Funkin-android#1617] [mods]
Download link: https://drive.google.com/file/d/1GvXaZWDZeAjZSTV73oJ-ElY-sx5rTSLM/view?usp=drivesdk

Screenshot:
![Screenshot_20210707-132322](https://user-images.githubusercontent.com/83911327/124711833-bdbe8e00-df28-11eb-87e7-ad84f87a5b65.jpg)
![Screenshot_20210707-093609](https://user-images.githubusercontent.com/83911327/124712362-6d93fb80-df29-11eb-8324-7a251dfcfe16.jpg)
Enjoy!

**Comment by StrYk85** [comment]
Wait you made it?
The mod,not the port

**Reply by DzGithub2805** [reply]
No, i did not made the mod, i port it
Original: https://youtu.be/5dPlEXgwtwQ
